### DISCUSSION TITLE
VDH license issue

### DISCUSSION THREAD

**Discussion by VinceB662** [aclap-dev/video-downloadhelper#1276] [License & Purchase requests]
### Your question

Had to refresh / reload corrupted Firefox browser. How do I re verify my VDH license
Also, I noticed in the past, VDH would not detect media when Norton VPN is deployed. I would have to disable Norton VPN
in order for media to be detected. Any reason why?
Please advise on both issues.
Thanks in advance for your help

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "download_directory": "C:\\Users\\vberl\\Documents\\My Recordings\\You Tube Downloads\\You Tube Download Misc",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 3,
    "successfull_dl": 1
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Let's assume you kept the original E-mail you received when you purchased your VDH license.  Just copy/paste your license from that E-mail into the VDH settings, same way you did the first time.

I know nothing about your VPN so I can't help you with that.

**Reply by VinceB662** [reply]
Found the solution by clicking validate license after license entry.
Case closed.

**Comment by VinceB662** [comment]
Thanks for your response. I have my original E-mail with the license info. I tried several times to put it into the VDH settings but it still doesn't work.

    On Tuesday, September 17, 2024 at 08:41:40 PM EDT, GrampaWildWilly ***@***.***> wrote:   

 


I do not work for VDH. I'm just another user.

Let's assume you kept the original E-mail you received when you purchased your VDH license. Just copy/paste your license from that E-mail into the VDH settings, same way you did the first time.

I know nothing about your VPN so I can't help you with that.
